MagicalRecord/Categories/DataImport/NSEntityDescription+MagicalDataImport.h

```diff
@@ -7,11 +7,12 @@
 //
 
 #import <CoreData/CoreData.h>
+#import "MagicalRecordXcode7CompatibilityMacros.h"
```

Comment:
>> And that's how it is in MagicalRecord 2.3.0, so :+1:

Not in this pull request. Because of the preprocessor.
Suppose, I import an umbrella header which contains

```obj-c
#import <MagicalRecord/NSEntityDescription+MagicalDataImport.h>
```

The user's file will be pre-processed to :
```obj-c

#import <CoreData/CoreData.h>
#import "MagicalRecordXcode7CompatibilityMacros.h"

// Declarations from NSEntityDescription+MagicalDataImport.h
```